### PR TITLE
tasks code and task history displayed in app panel

### PR DESCRIPTION
**Pull request by deepakdinesh1123**

### CODE REVIEW

**Comment by kc-diabeat** [comment]
The build artifact is available in the artifact section [here](https://github.com/Healthlane-Technologies/Zango/actions/runs/10386570388). You can download it and place the build file under backend/src/zango/assets/app_panel/js for local testing and verification.

**Comment by kc-diabeat** [comment]
The build artifact is available in the artifact section [here](https://github.com/Healthlane-Technologies/Zango/actions/runs/10632136564). You can download it and place the build file under backend/src/zango/assets/app_panel/js for local testing and verification.

**Comment by kc-diabeat** [comment]
The build artifact is available in the artifact section [here](https://github.com/Healthlane-Technologies/Zango/actions/runs/10632179350). You can download it and place the build file under backend/src/zango/assets/app_panel/js for local testing and verification.

**Comment by kc-diabeat** [comment]
The build artifact is available in the artifact section [here](https://github.com/Healthlane-Technologies/Zango/actions/runs/10662713793). You can download it and place the build file under backend/src/zango/assets/app_panel/js for local testing and verification.

File: backend/src/zango/assets/app_panel/js/build.1723531926721.min.js.LICENSE.txt
Comment (shahharsh176) [inline backend/src/zango/assets/app_panel/js/build.1723531926721.min.js.LICENSE.txt:1]: @prem056627 Please remove this file

File: backend/src/zango/assets/app_panel/js/build.1723136204750.min.js
Comment (shahharsh176) [inline backend/src/zango/assets/app_panel/js/build.1723136204750.min.js:1]: @prem056627 please revert this change

File: backend/src/zango/assets/js/jquery/3.7.1/jquery.min.js
Comment (shahharsh176) [inline backend/src/zango/assets/js/jquery/3.7.1/jquery.min.js:1]: @prem056627 Please revert this change

File: frontend/src/metadata.json [hunk 1]
```diff
@@ -1 +1 @@
-{ "buildMajor": 0, "buildMinor": 3, "buildPatch": 0, "buildTag": "" }
+{"buildMajor":0,"buildMinor":3,"buildPatch":1,"buildTag":""}
```
Comment (shahharsh176) [inline frontend/src/metadata.json:1]: @prem056627 Please revert changes of this file

**Review by shahharsh176 — CHANGES_REQUESTED** [review]

**Comment by kc-diabeat** [comment]
The build artifact is available in the artifact section [here](https://github.com/Healthlane-Technologies/Zango/actions/runs/10697131714). You can download it and place the build file under backend/src/zango/assets/app_panel/js for local testing and verification.

**Comment by kc-diabeat** [comment]
The build artifact is available in the artifact section [here](https://github.com/Healthlane-Technologies/Zango/actions/runs/10698836624). You can download it and place the build file under backend/src/zango/assets/app_panel/js for local testing and verification.

File: backend/src/zango/assets/app_panel/js/build.1723531926721.min.js.LICENSE.txt
Comment (shahharsh176) [inline backend/src/zango/assets/app_panel/js/build.1723531926721.min.js.LICENSE.txt:1]: remove this file

File: backend/src/zango/assets/js/jquery/3.7.1/jquery.min.js
Comment (shahharsh176) [inline backend/src/zango/assets/js/jquery/3.7.1/jquery.min.js:1]: please revert this change

File: backend/src/zango/assets/app_panel/js/build.1723136204750.min.js
Comment (shahharsh176) [inline backend/src/zango/assets/app_panel/js/build.1723136204750.min.js:1]: please revert this change

**Review by shahharsh176 — CHANGES_REQUESTED** [review]

File: backend/src/zango/api/platform/tasks/v1/views.py [hunk 2]
```diff
@@ -70,6 +78,7 @@ def post(self, request, app_uuid, *args, **kwargs):
             tenant = TenantModel.objects.get(uuid=app_uuid)
             connection.set_tenant(tenant)
             with connection.cursor() as c:
+                ws = Workspace.get_plugin_source()
```
Comment (shahharsh176) [inline backend/src/zango/api/platform/tasks/v1/views.py:81]: @deepakdinesh1123 Is this needed over here? because below we are creating new Workspace instance

File: backend/src/zango/assets/app_panel/js/build.1723531926721.min.js.LICENSE.txt
Comment (prem056627) [reply backend/src/zango/assets/app_panel/js/build.1723531926721.min.js.LICENSE.txt:1]: removed

**Comment by kc-diabeat** [comment]
The build artifact is available in the artifact section [here](https://github.com/Healthlane-Technologies/Zango/actions/runs/10701590004). You can download it and place the build file under backend/src/zango/assets/app_panel/js for local testing and verification.

**Comment by kc-diabeat** [comment]
The build artifact is available in the artifact section [here](https://github.com/Healthlane-Technologies/Zango/actions/runs/10702184120). You can download it and place the build file under backend/src/zango/assets/app_panel/js for local testing and verification.

**Comment by kc-diabeat** [comment]
The build artifact is available in the artifact section [here](https://github.com/Healthlane-Technologies/Zango/actions/runs/10702297671). You can download it and place the build file under backend/src/zango/assets/app_panel/js for local testing and verification.

File: frontend/src/pages/appTaskManagement/components/AppTaskManagement/index.js [hunk 3]
```diff
@@ -101,6 +102,7 @@ export default function AppTaskManagement() {
 			</div>
 			<UpdateTaskModal />
 			<RemoveAllPoliciesModal />
```
Comment (sukuna28) [inline frontend/src/pages/appTaskManagement/components/AppTaskManagement/index.js:104]: @prem056627  can you check if this RemoveAllPoliciesModal is required, if not remove it

File: frontend/src/pages/appTaskManagement/components/AppTable/TaskHistoryModal/TaskHistoryContainer.jsx [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+    console.log(TaskHistoryData);
+
+    const handleToggleCode = (index) => {
+        setVisibleCodeIndex(visibleCodeIndex === index ? null : index);
+    };
+
+    return (
+        <div className="overflow-y-auto">
+           {console.log('TaskHistoryData',TaskHistoryData)}
```
Comment (sukuna28) [inline frontend/src/pages/appTaskManagement/components/AppTable/TaskHistoryModal/TaskHistoryContainer.jsx:17]: @prem056627  remove console log

**Review by sukuna28 — CHANGES_REQUESTED** [review]

**Comment by kc-diabeat** [comment]
The build artifact is available in the artifact section [here](https://github.com/Healthlane-Technologies/Zango/actions/runs/10767183086). You can download it and place the build file under backend/src/zango/assets/app_panel/js for local testing and verification.

File: frontend/src/pages/appTaskManagement/components/AppTable/TaskHistoryModal/TaskHistoryContainer.jsx (hunk 4 above)
Comment (shahharsh176) [reply frontend/src/pages/appTaskManagement/components/AppTable/TaskHistoryModal/TaskHistoryContainer.jsx:15]: @prem056627 Can you please remove this ?

**Comment by kc-diabeat** [comment]
The build artifact is available in the artifact section [here](https://github.com/Healthlane-Technologies/Zango/actions/runs/10811658972). You can download it and place the build file under backend/src/zango/assets/app_panel/js for local testing and verification.

**Review by shahharsh176 — APPROVED** [review]
